sys-apps/acl/files/acl-2.3.1-musl-1.2.4-lfs64-fix.patch

```diff
@@ -0,0 +1,37 @@
+Bug: https://bugs.gentoo.org/905910
+Upstream Bug: https://savannah.nongnu.org/bugs/index.php?64162
+
+From a9100afd77fea00b311f114a5a04108283aa681a Mon Sep 17 00:00:00 2001
+From: Violet Purcell <vimproved@inventati.org>
+Date: Mon, 8 May 2023 04:17:07 +0000
+Subject: [PATCH] musl 1.2.4 LFS64 removal fixes
+
+---
+ tools/chacl.c | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/tools/chacl.c b/tools/chacl.c
+index 525a7ff..8fff875 100644
+--- a/tools/chacl.c
++++ b/tools/chacl.c
+@@ -320,7 +320,7 @@ walk_dir(acl_t acl, acl_t dacl, const char *fname)
+ {
+ 	int failed = 0;
+ 	DIR *dir;
+-	struct dirent64 *d;
++	struct dirent *d;
```

Comment:
Some configure logic may be needed for glibc to ensure that these are 64-bit.

i.e.
* -D_LARGEFILE_SOURCE makes dirent64 available, but keeps dirent 32-bit
* -D_FILE_OFFSET_BITS=64 makes dirent 64-bit

Is AC_SYS_LARGEFILE in configure.ac? If not, put it in. It should be fine then.

Reply:
Got it, will check

Reply:
thanks!